synapse/lib/modelrev.py

```diff
@@ -658,6 +658,8 @@ async def revModel20230209(self, layers):
 
         meta = {'time': s_common.now(), 'user': self.core.auth.rootuser.iden}
 
+        await self._normFormSubs(layers, 'inet:http:cookie')
```

Comment:
Probably doesn't matter but should move this before `meta` is set for the other edits here so the time isn't out of order